apps/super-app/src/components/navbar/NavBar.css

```diff
@@ -0,0 +1,38 @@
+@import url('https://fonts.googleapis.com/css?family=Hind');
+a {
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 32px 10px;
```

Comment:
What's the reasoning behind these values? It doesn't seem necessary for centering.

Reply:
Hi Glen! 
The padding is so that there is a spacing between the icons at the edges and the sides of the box (it will look like the pic i attached below). However, I understand that using px for the padding is not scalable, I was wondering if there is another way to design this? Thanks!
![image](https://user-images.githubusercontent.com/92622007/212688378-75b5ef35-bc2f-40b3-aa94-e02d9dd762e0.png)

Reply:
Ah I meant vertical centering. The horizontal padding is fine; I don't understand the discrepancy between top and bottom padding.

The simple way to avoid px is to use `rem` :) you can eyeball the values and it should be fine. Don't feel the need to replace px with other units at this time though, we can adjust them later.